hirundo/_iter_sse_retrying.py

```diff
@@ -44,13 +48,26 @@ def _iter_sse():
             connect_headers["Last-Event-ID"] = last_event_id
 
         with connect_sse(client, method, url, headers=connect_headers) as event_source:
-            for sse in event_source.iter_sse():
-                last_event_id = sse.id
-
-                if sse.retry is not None:
-                    reconnection_delay = sse.retry / 1000
-
-                yield sse
+            try:
+                for sse in event_source.iter_sse():
+                    last_event_id = sse.id
```

Comment:
I feel out of the loop.. what is sse?

Reply:
it keeps popping all over

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events/Using_server-sent_events
(HTTP Server-Sent Events)